internal/firewall/iptables.go

```diff
@@ -179,6 +179,17 @@ func (c *Config) acceptOutputFromIPToSubnet(ctx context.Context,
 	return c.runIP6tablesInstruction(ctx, instruction)
 }
 
+func (c *Config) acceptIpv6MulticastOutput(ctx context.Context,
```

Comment:
Can you add a comment above the function definition why this is needed? Sorry I'm a bit of an ipv6 noob..